streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorStateManager.java

```diff
@@ -205,7 +212,8 @@ public ProcessorStateManager(final TaskId taskId,
                                  final ChangelogRegister changelogReader,
                                  final Map<String, String> storeToChangelogTopic,
                                  final Collection<TopicPartition> sourcePartitions,
-                                 final boolean stateUpdaterEnabled) throws ProcessorStateException {
+                                 final boolean stateUpdaterEnabled,
+                                 final boolean startupState) throws ProcessorStateException {
```

Comment:
What does `startupState` represent? Would it be better to use an `enum` in this case?
If that's not appropriate, then some brief comments on what the meaning of `true` and `false` represent.